npm-update.js

```diff
@@ -1,189 +1,88 @@
 /* Copyright (c) 2014-2015 Richard Rodger, MIT License */
 /* jshint node:true, asi:true, eqnull:true */
-"use strict";
+"use strict"
 
 
-var follow = require('follow');
-var npm_stats = require('npm-stats')
-var async     = require('async')
+var follow     = require('follow')
+var NpmStats   = require('npm-stats')
+var async      = require('async')
+var JSONStream = require('JSONStream')
 
 
-module.exports = function npm_update( options ){
+module.exports = function npmUpdate( options ){
   var seneca = this
 
-  options = seneca.util.deepextend({
-    task: null,
-    batchsize: 33,
+  var opts = Object.assign({
```

Comment:
Curiosity question -- what does seneca deep extend do? I assumed this might have been put into place before es6, but now that we have object.assign, and it is more than likely optimized, shouldn't we use it? I'll change for consistency.